corehq/util/urlsanitize/test/test_urlsanitize.py

```diff
@@ -40,10 +42,8 @@ def setup_module():
 
 def test_example_suite():
     for input_url, (result, value) in SUITE:
-        print(input_url)
-
         if result is RETURN:
-            eq(sanitize_user_input_url(input_url), value)
+            sanitize_user_input_url(input_url)
```

Comment:
Are tests that use this branch only verifying that an exception is not raised? Is it not worth testing anything about the return value, such as that it is an instance of `IPv4Address` or something like that?

Reply:
This coincides with the other comment you made, but the issue was that this function was misnamed. It's a validation function, so the absence of an exception is the only thing we're testing here.